Interpreting-GitHubActions-Results.md

```diff
@@ -0,0 +1,16 @@
+# Interpreting GitHub Actions Results
+
+## Detailed Explanation
+
+Our workflow tests run in parallel as to alleviate the overall pressure on the queue.
+
+  ![GHCI Status](images/ghciSample.png)
+
+In the example above, the lint checks and Mypy checks have failed. However, the rest of the tests have passed.
+
+We also have two checks that have been skipped due to the failure of one of the e2e tests. In the event of an e2e test failure, any remaining e2e tests that are queued or currently running will be terminated and marked as "skipped."
+
+### What should you do?
+
+- Following the example above, you must first fix the [failing lint checks, mypy checks and e2e tests](https://github.com/oppia/oppia/wiki/If-CI-checks-fail-on-your-PR#failing-tests-and-lint-checks) then push.
+- Go to 'Details' to view the test logs.
```

Comment:
Let's add images here of where to find the logs

Reply:
Is another image necessary though? The first image already shows the 'Details' section and I believe most beginners with GitHub actions know where to find the logs.

Reply:
@deadex-ng I think image will be of help here as most of the new contributors are new to github too and don't know much about GitHub actions, so adding a image as @gp201 suggested would be of help for such contributors.